### ISSUE TITLE
--oracles-resolver - unexpected argument on nargo prove 0.22.0

### ISSUE THREAD

**Issue by LogvinovLeon** [noir-lang/noir#4096]
### Aim

I tried to run a proof for a circuit that uses oracles using nargo. The docs describe the `--oracles-resolver` argument being available on 0.22.0 version

### Expected Behavior

It uses the provided argument to fetch oracle data

### Bug

```sh
nargo prove --oracle-resolver=http://localhost:5555
error: unexpected argument '--oracle-resolver' found

Usage: nargo prove [OPTIONS]

For more information, try '--help'.
```

### To Reproduce

1. Install nargo latest using noirup
2. Create an empty noir project
3. Run `nargo prove --oracle-resolver=localhost`
4. See an issue

### Installation Method

Binary

### Nargo Version

0.22.0+3fae4a03fded4e3f5065e7461c563f7e39745604

### Would you like to submit a PR for this Issue?

Maybe

**Comment by LogvinovLeon** [comment]
My bad. I was looking at nightly version docs. Oracle resolver is not yet available on 0.22.0. Closing

**Comment by TomAFrench** [comment]
Reopening this as I can see references to resolving oracles in the v0.22.0 docs. We should remove references to nargo being able to resolve these as this is only supported from v0.23.0, references to noir_js can stay however.

**Comment by TomAFrench** [comment]
@noir-lang/developerrelations

**Comment by critesjosh** [comment]
@TomAFrench so these pages should be removed from 0.22 docs?
- https://noir-lang.org/docs/explainers/explainer-oracle
- https://noir-lang.org/docs/how_to/how-to-oracles
- https://noir-lang.org/docs/noir/syntax/oracles

**Comment by signorecello** [comment]
They should. I actually added oracles manually to the 0.22.0 docs.

I realize now that probably @Savio-Sou and me were wrong in thinking they were already available for 0.22.0.

I'll remove these pages and leave the correct ones, on the dev version
